aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -1204,7 +1204,7 @@ Tensor cudnn_convolution_backward_weight(
   // Make sure that NC11 strides follow formula
   grad_output_contig_t.resize_(grad_output_contig_t.sizes(), layout);
   TensorArg grad_output_contig{ grad_output_contig_t, "grad_output", 1 };
- 
+
```

Comment:
Can you undo this change? It does not look to be related to the rest.

Reply:
which one do you want me to undo? or are we good for now?